### PR TITLE
Update Portfolio Section

### PR DESCRIPTION
**Pull request by zediogoviana**
Updated portfolio section.

Why: 

- For website rebranding.

How:

- Mobile and Desktop versions according to the design. However, due to `WavyLink` limitations, the inverted versions (**wave-link** and not **link-wave**) and underlined links were not developed (necessary to alter `WavyLink` and `Typography` components first).

### CODE REVIEW

**Comment by filipebfernandes** [comment]
1. On Desktop the siosLife wave should be on the left of the viewport with the text and wave reverted

1. siosLife services copy are the ones from lyftonomie.

1. On mobile, the wave is too small

**Comment by zediogoviana** [comment]
@filipebfernandes I didn't revert the siosLife wave because it is necessary to alter the **WavyLink** component, and I was not sure if I should do it in this PR or in a separate one.

**Comment by filipebfernandes** [comment]
> @filipebfernandes I didn't revert the siosLife wave because it is necessary to alter the **WavyLink** component, and I was not sure if I should do it in this PR or in a separate one.

Do as you must 😄 If you need to change them for this PR do it in this one.

File: src/sections/HomePortfolio/index.css [hunk 1]
```diff
@@ -1,300 +1,284 @@
...
 }
 
-.project {
-  display: flex;
-  flex-direction: column;
-  width: 100%;
-
-  margin-top: $spacing-224;
+.project-grid {
+  display: grid;
```
Comment (taniaprocha) [inline src/sections/HomePortfolio/index.css:11]: As I've explained in our meeting, we preferentially use flex. To change order of columns with flex you can use order, check services section if you need.

File: src/sections/HomePortfolio/index.css [hunk 2]
```diff
@@ -1,300 +1,284 @@
...
 
-.title > *:last-child {
-  margin-top: $spacing-20;
+.project-column-left {
+  position: absolute;
+
+  grid-column: 1 / span 1;
+  display: flex;
+  flex-direction: column;
+  /*width: 100%;*/
```
Comment (taniaprocha) [inline src/sections/HomePortfolio/index.css:52]: 💣

**Comment by taniaprocha** [comment]
Yah, change `WavyLink` on this PR.

File: src/components/WavyLink/index.js [hunk 3]
```diff
@@ -49,6 +79,7 @@ WavyLink.propTypes = {
   label: PropTypes.string,
   animate: PropTypes.bool,
   target: PropTypes.oneOf(["blank", "self"]),
+  position: PropTypes.string,
```
Comment (taniaprocha) [inline src/components/WavyLink/index.js:82]: Please add `.oneOf...` with the possible values for position

**Comment by zediogoviana** [comment]
@filipebfernandes I altered the things you mentioned, you can check them now. 
@taniaprocha the move from grid to flexbox is also done.

**Comment by taniaprocha** [comment]
Thanks @zediogoviana for your help. Good work. I'm going to proceed with this PR.

**Comment by filipebfernandes** [comment]
Looking good 👏 Noticed that `Full Stack Development` is on separate lines. It should be on the same.

**Pull request merged**